crates/oxc_transformer/src/react_jsx/mod.rs

```diff
@@ -27,6 +27,8 @@ pub struct ReactJsx<'a> {
     import_jsxs: bool,
     import_fragment: bool,
     import_create_element: bool,
+    import_source: String,
```

Comment:
We can probably save a allocation with `Cow<'static, str>`

Reply:
This is duplicated across `Transformer` and `TransformOptions`, I think it's not necessary to copy it over here.